README.md

```diff
@@ -29,6 +29,10 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
   - admin:org (read:org)
   - user (read:user, user:email)
 
+### Optional (GitOps operators)
+  If you plan to deploy in a GitOps environment, make sure you have installed the `ArgoCD/Red Hat OpenShift GitOps` and the `Tekton/Red Hat Openshift Pipelines Install` operators following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/gitops/README.md).
+  The Orchestrator installs RHDH and imports software templates designed for bootstrapping workflow development. These templates are crafted to ease the development lifecycle, including a Tekton pipeline to build workflow images and generate workflow K8s custom resources. Furthermore, ArgoCD is utilized to monitor any changes made to the workflow repository and to automatically trigger the Tekton pipelines as needed. This installation process ensures that all necessary Tekton and ArgoCD resources are provisioned within the same cluster.
+
 ## Installation
 
 1. Get the Helm chart from one of the following options
```

Comment:
Since you removed the Manual Chart, then there is only one option :) 
Can you please reword this sentence?

Reply:
Done